code/modules/mob/living/carbon/alien/humanoid/queen.dm

```diff
@@ -30,7 +31,7 @@
 
 /mob/living/carbon/alien/humanoid/queen/movement_delay()
 	. = ..()
-	. += 3
+	. += 2

```

Comment:
um you still increased the speed of the queen

Reply:
That's intentional.